src/neuroconv/utils/json_schema.py

```diff
@@ -21,15 +21,10 @@ def default(self, obj):
         if isinstance(obj, datetime):
             return obj.isoformat()
 
-        # This should transforms numpy generic integers and floats to python floats
+        # Transform numpy generic integers and floats to python ints floats
         if isinstance(obj, np.generic):
             return obj.item()
 
-        # Numpy-versions of various numeric types
-        if isinstance(obj, np.integer):
-            return int(obj)
-        if isinstance(obj, np.floating):
-            return float(obj)
```

Comment:
The above line that handles generic types handles all float and integer types, so these lines are redundant